src/drf_yasg/inspectors.py

```diff
@@ -114,7 +114,8 @@ def get_view_serializer(self):
         """
         if not hasattr(self.view, 'get_serializer'):
             return None
-        return self.view.get_serializer()
+        kwargs = {'context': self.view.get_serializer_context()}
```

Comment:
In fact, this line is exactly like the line you linked to, and it is in the same function that is called above.